docs/mkdocs.base.yml

```diff
@@ -1,6 +1,6 @@
-site_name: My Docs
+site_name: Quantum Ready Solution for Drug Discovery
 site_url: https://example.com/
```

Comment:
update site_url to `https://awslabs.github.io/quantum-ready-solution-for-drug-discovery/`